lib/src/fields/form_builder_signature_pad.dart

```diff
@@ -9,9 +9,9 @@ import 'package:signature/signature.dart';
 class FormBuilderSignaturePad extends StatefulWidget {
   final String attribute;
   final List<FormFieldValidator> validators;
-  @Deprecated(
+/*   @Deprecated(
```

Comment:
It appears that you managed to make `initialValue` work, so that is why you removed this Deprecation message.  Rather than just commenting it out, I suggest removing it altogether.  (It otherwise becomes _litter_.)

Reply:
Done